centreon/www/install/php/Update-next.php

```diff
@@ -120,6 +121,209 @@
+            AND acl_topology_relations.topology_topology_id IN (
+                SELECT topology_id FROM topology WHERE topology_page IN (60801, 60802, 60803, 60807)
+            )
+            SQL,
+        QueryParameters::create([
+            QueryParameter::int('acl_topo_id', $aclTopologyId),
+        ])
+    );
+};
+
+$insertNewCommandsTopologyRights = function (int $aclTopologyId) use ($pearDB, &$errorMessage): void {
+    $errorMessage = 'Unable to insert into table acl_topology_relations';
+    $pearDB->executeStatement(
+        <<<'SQL'
+            INSERT INTO acl_topology_relations (acl_topo_id, topology_topology_id, access_right)
+            VALUES (:acl_topo_id, :topology_topology_id, :access_right)
+            SQL,
+        QueryParameters::create([
+            QueryParameter::int('acl_topo_id', $aclTopologyId),
+            QueryParameter::int('topology_topology_id', 60808),
+            QueryParameter::int('access_right', 1),
+        ])
+    );
+};
+
+$getOrCreateActionGroup = function (int $aclGroupId, array &$actionGroupRelations) use ($pearDB, &$errorMessage): ?array {
+
+    $actionGroup = null;
+    foreach ($actionGroupRelations as $relation) {
+        if ($relation['acl_group_id'] === $aclGroupId) {
+            $actionGroup = $relation;
+            break;
+        }
+    }
+
+    if ($actionGroup === null) {
+        $errorMessage = 'Unable to create a new acl_action';
+        $pearDB->executeStatement(
+            <<<'SQL'
+                INSERT INTO acl_actions (acl_action_name, acl_action_activate)
+                VALUES (CONCAT((SELECT acl_group_name FROM acl_groups WHERE acl_group_id = :acl_group_id), '_actions'), '1')
+                SQL,
+            QueryParameters::create([
+                QueryParameter::int('acl_group_id', $aclGroupId),
+            ])
+        );
+        $actionId = (int) $pearDB->lastInsertId();
+
+        $errorMessage = 'Unable to link a new acl_action to an acl_group';
+        $pearDB->executeStatement(
+            <<<'SQL'
+                INSERT INTO acl_group_actions_relations (acl_group_id, acl_action_id)
+                VALUES (:acl_group_id, :acl_action_id)
+                SQL,
+            QueryParameters::create([
+                QueryParameter::int('acl_group_id', $aclGroupId),
+                QueryParameter::int('acl_action_id', $actionId),
+            ])
+        );
+        $actionGroup = [
+            'acl_group_id' => $aclGroupId,
+            'acl_action_id' => $actionId,
+        ];
+        $actionGroupRelations[] = $actionGroup;
+    }
+
+    return $actionGroup;
+};
+
+$addCommandRightIntoAction = function (string $commandType, int $accessRight, int $aclActionId) use ($pearDB, &$errorMessage): void {
+    if ($accessRight === 0) {
+        return;
+    }
+    $actionName = match($accessRight) {
+        1 => "manage_{$commandType}_commands",
+        2 => "see_{$commandType}_commands",
+        default => null,
+    };
```

Comment:
```suggestion
    $actionName = match($accessRight) {
        1 => "manage_{$commandType}_commands",
        2 => "see_{$commandType}_commands",
        default => null,
    };
    
    if ($actionName === null) {
        return;
    }
```

Only if the null value is not possible after in the SELECT / INSERT Queries (And I think the default is here for coding style purpose but should not occurs in real case)